runtime/src/bank.rs

```diff
@@ -7045,6 +7044,18 @@ impl TransactionProcessingCallback for Bank {
     }
 }
 
+impl fmt::Debug for Bank {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Bank")
+            .field("slot", &self.slot)
+            .field("bank_id", &self.bank_id)
+            .field("block_height", &self.block_height)
+            .field("parent_slot", &self.parent_slot)
+            .field("capitalization", &self.capitalization())
+            .finish_non_exhaustive()
```

Comment:
would stuff like the various hashes and leader id be valuable too?

Reply:
we can let people who are mad their favorite field is missing add them as they see fit